src/main/java/com/cicdlectures/menuserver/controller/MenuController.java

```diff
@@ -37,4 +42,10 @@
   public MenuDto createMenu(@RequestBody MenuDto menu) {
     return createMenuService.createMenu(menu);
   }
+
+  @DeleteMapping(path = "/menus/{id}")
+  int deleteMenu(@PathVariable Long id) {
+    this.menuRepository.deleteById(id);
+    return 200;
```

Comment:
Pour la réponse http, il faudrait plus utiliser @ResponseStatus à mon avis.

Reply:
will work on it